doc/0001-record-architecture-decisions.md

```diff
@@ -0,0 +1,91 @@
+# 1. Record architecture decisions
```

Comment:
Sono solo in dubbio sulla lingua. Dato che ha senso usare l'italiano nella documentazione, tradurrei anche i titoli delle sezioni in italiano